lua/neogit/popups/rebase/actions.lua

```diff
@@ -115,6 +115,24 @@ M.reword = operation("rebase_reword", function(popup)
   git.rebase.reword(commit, new_message)
 end)
 
+M.modify = operation("rebase_modify", function(popup)
+  local commit
+  if popup.state.env.commit then
+    commit = popup.state.env.commit
+  else
+    commit = CommitSelectViewBuffer.new(git.log.list()):open_async()[1]
+    if not commit then
+      return
+    end
+  end
+  local message = git.log.message(commit)
+  if
+    input.get_confirmation('Modify commit "' .. message .. '"?', { values = { "&Yes", "&No" }, default = 1 })
```

Comment:
I'm not sure we need to ask for confirmation here - the user has already hit `rm` and (possibly) chosen a commit.

Reply:
Fair enough, I removed it.